trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts

```diff
@@ -29,8 +29,8 @@ describe(page, () => {
     reportId: reportId.toString(),
     feeRecordPaymentGroups: [],
     enablePaymentsReceivedSorting: false,
-    premiumPaymentFormError: undefined,
-    facilityIdQueryError: undefined,
+    tableError: undefined,
+    filterError: undefined,
```

Comment:
Why do we need `tableError`? `filterError` seems to make sense, but unless i'm missing something, it looks like "table error handling" could be simpler - assuming the table consumes an array of objects, feels like the UI could check if the array is populated and if not render an error? Instead of `getGenerateKeyingDataError` and `generateKeyingDataErrorMap`

Reply:
Ignore my previous comment I can see there are multiple error scenarios which I assume will render a bespoke error messsage in the UI

```ts
case 'no-fee-records-selected':
case 'different-fee-record-statuses':
case 'different-fee-record-payment-currencies':
case 'multiple-does-not-match-selected':
```

Do you think it should be called maybe `tableDataError` instead of `tableError`? As it feels like "no data for the table to consume" rather than an issue with the table itself

Reply:
Thanks for renaming 🎉